src/platforms/react-native/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -22,6 +22,12 @@ export default Sentry.wrap(App);
 
 ### Enable Routing Instrumentation
 
+<Note>
+
+When no routing instrumentation is used, a transaction for `App Start` is automatically captured. That stops being sent when one of the routing integrations below is added. That information lives as a Span in a transaction captured by the routing instrumentation instead.
```

Comment:
@imatwawana It's likely I didn't explain this really well. Do you think this is OK?

Reply:
I think it's pretty good. What about this?

```suggestion
When no routing instrumentation is used, a transaction for `App Start` is automatically captured. However, that transaction stops being sent when one of the routing integrations below is added. Instead, the `App Start` information is included as a span in a transaction captured by the routing instrumentation.
```